lib/src/analyzer.dart

```diff
@@ -17,7 +17,16 @@ export 'package:analyzer/src/generated/resolver.dart'
     show InheritanceManager, TypeProvider, TypeSystem;
 export 'package:analyzer/src/generated/source.dart' show LineInfo, Source;
 export 'package:analyzer/src/lint/linter.dart'
-    show DartLinter, LintRule, Group, Maturity, LinterOptions, LintFilter;
+    show
+        DartLinter,
+        LintRule,
+        Group,
+        Maturity,
+        LinterOptions,
+        LintFilter,
+        NodeLintRegistry,
+        NodeLintRule,
+        UnitLintRule;
```

Comment:
Should we have `UnitLintRule`? It's only being used in two places, and we could almost as easily just register to process a `CompilationUnit`. It's not clear to me that it's worth the weight of have an alternate interface to implement to save a three line method in two places.